web_ui/src/useApi.ts

```diff
@@ -50,3 +55,13 @@ export function useTeamApi<T>(
 
   return state
 }
+
+export function teamApi<T, V extends ITeamArgs>(
+  func: (args: V) => Promise<T>,
+  args: Omit<V, "teamId"> = {} as Omit<V, "teamId">,
```

Comment:
we're skirting around the type system when we cast the `{}` to `Omit<V, "teamId">`

If `V` has extra properties that `func` relies on then it will break at run time

Reply:
is this used anywhere?

Reply:
it's used a lot of the UsageBillingPage.tsx. Do you have a suggested fix?